content/en/docs/contribution-guidelines/upgrade-graphs.md

```diff
@@ -0,0 +1,241 @@
+---
+
+By checking this document, you will be able to understand the legends used in the Channel Upgrade Graphs as how to create them. Note that it also can be useful to drawing your own graphs to represent your scenarios if you need a help with and communicate with others. 
+
+## Legends
+
+| <div style="align-text: center;width:200px">Diagram (e.g)</div> | Description |   
+|----------|:-------------:|
+| {{<mermaid>}}
+flowchart TB
+   classDef head fill:#ffbfcf;
+   classDef installed fill:#34ebba;
+   
+   ID(v0.0.1):::head
+{{</mermaid>}} | Operator bundle version which is the head of the channel |
+| {{<mermaid>}}
+flowchart TB
+   classDef head fill:#ffbfcf;
+   classDef installed fill:#34ebba;
+   
+   ID(v0.0.1):::installed
+{{</mermaid>}} | Operator bundle version which is installed currently |
+| {{<mermaid>}}
+flowchart TB
+   classDef head fill:#ffbfcf;
+   classDef installed fill:#34ebba;
+   
+   ID(v0.0.1)
+{{</mermaid>}} | Operator bundle version which is installable |
+| {{<mermaid>}}
+flowchart TB
+   classDef head fill:#ffbfcf;
+   classDef installed fill:#34ebba;
+   
+   subgraph preview
+      ID(v0.0.1) 
+   end
+{{</mermaid>}} | Operator bundle channel. (eg. from the bundle `annotations.yaml`, `operators.operatorframework.io.bundle.channels.v1: preview` ) |
+| {{<mermaid>}}
+flowchart TB
+   classDef head fill:#ffbfcf;
+   classDef installed fill:#34ebba;
+   
+   subgraph "preview (default)"
+      ID(v0.0.1) --> ID2(v0.0.2)
+   end
+{{</mermaid>}} | Operator default channel. (eg. from the bundle `annotations.yaml`, `operators.operatorframework.io.bundle.channel.default.v1: stable` ) |
+| {{<mermaid>}}
+flowchart TB
+   classDef head fill:#ffbfcf;
+   classDef installed fill:#34ebba;
+   
+   A(v0.0.1) --> B(v0.0.4)
+{{</mermaid>}} | An upgrade path to replace one operator bundle version for another using the [CSV][csv-definition] `replaces spec. (eg.`spec.replaces: exampleoperator.v0.0.1`). More info: [here][upgrade-path-replaces]. |
+| {{<mermaid>}}
+flowchart TB
+   classDef head fill:#ffbfcf;
+   classDef installed fill:#34ebba;
+   
+   A(v0.0.1) x--x |v0.0.2,v0.0.3| B(v0.0.4)
+{{</mermaid>}} | An upgrade path to skip versions in the upgrade path using the [CSV][csv-definition] `skips` spec. (eg.`spec.skips: exampleoperator.v0.0.2, exampleoperator.v0.0.3`). More info: [here][upgrade-path-skips]. |
+| {{<mermaid>}}
+flowchart TB
+   classDef head fill:#ffbfcf;
+   classDef installed fill:#34ebba;
+   
+   C(v2.0.3) o--o |>= 2.0.4 < 3.0.0| D(v3.0.1)
+{{</mermaid>}} |  An upgrade path to skip a range of operator bundle versions using the [CSV][csv-definition] `olm.skipRange` annotation. (E.g `annotations.olm.skipRange: '>= 2.0.4 < 3.0.0'`) More info: [here][upgrade-path-skiprange]. |
+| {{<mermaid>}}
+flowchart TB
+   classDef head fill:#ffbfcf;
+   classDef installed fill:#34ebba;
+   
+   A(v0.0.1) -.-> B(v0.0.4)
+{{</mermaid>}} | Represent the same replace method describe above but for a future scenario |
```

Comment:
Same here, it'd be nice to talk about what it means for the graph, without mentioning OLM internals